modules/storage-multi-attach-error.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-storage-issues.adoc
+
+[id="storage-multi-attach-error_{context}"]
+= Resolving multi-attach errors
+
+.Problem
+When a node crashes or shuts down abruptly, the ReadWriteOnce (RWO) volume is expected to be unmounted from the node so that it can be used by a Pod scheduled on another node.
+
+Mounting on a new node is not possible, however, because when the failed node is down, the system cannot unmount the attached volume.
```

Comment:
@bobfuru for this sentence, I would say that something like the following might be more accurate:

"Mounting on a new node is not possible, however, because the failed node was unable to properly unmount the attached volume."

Reply:
Although I do not fully understand the technical mechanism here, so please let me know if I am wrong.

Reply:
This makes sense to me, @pneedle-rh - thanks for the suggestion!